tools/ci/docker-containers-windows

```diff
@@ -152,6 +153,42 @@ case "$TARGET_CONTAINER" in
     fi
     ;;
 
+  alloy-devel-pr)
+	TAG="pr-${GITHUB_PR_NUMBER}-${VERSION_TAG}"
+
+    docker build                           \
+      -t "$DEVEL_ALLOY_IMAGE:$TAG" \
+      --build-arg VERSION="$VERSION"       \
+      --build-arg RELEASE_BUILD=1          \
+      --build-arg BASE_IMAGE_GO="$BASE_IMAGE_GO" \
+      --build-arg BASE_IMAGE_WINDOWS="$BASE_IMAGE_WINDOWS" \
+      -f ./Dockerfile.windows              \
+      .
+
+    if [[ $PUSH_ALLOY_IMAGE == "true"  ]]; then
+      docker push "$DEVEL_ALLOY_IMAGE:$TAG"
+    fi
+    ;;
+
+  alloy-devel-cngcrypto-pr)
```

Comment:
Is this built off of the `publish-dev:windows` tag?

Reply:
I removed `alloy-devel-cngcrypto-pr`. I can check in a follow up if we even need the parts for cngcrypto. As far as I can tell it's not being built and published at all atm